Cargo.toml

```diff
@@ -115,6 +115,7 @@ rustfmt-nightly = { path = "src/tools/rustfmt" }
 # See comments in `src/tools/rustc-workspace-hack/README.md` for what's going on
 # here
 rustc-workspace-hack = { path = 'src/tools/rustc-workspace-hack' }
+r-efi = { git = "https://github.com/r-efi/r-efi" }
```

Comment:
A new crates.io release will be needed before this can be merged.

Reply:
Yeah. Have already started the process on that: https://github.com/r-efi/r-efi/issues/47